packages/vue/src/components/molecules/SfRange/SfRange.vue

```diff
@@ -0,0 +1,87 @@
+<template>
+  <div ref="range" type="range" class="sf-range" :disabled="disabled">
+    <slot v-bind="$attrs" />
+  </div>
+</template>
+<script>
+import noUiSlider from "nouislider";
+export default {
+  name: "SfRange",
+  props: {
+    /*
+     * Sets the number of sliders (1 or 2) by adding initial values to the array
+     */
+    value: {
+      type: [Number, Array],
```

Comment:
Also, how do you expect to handle the case when someone passes number as a prop? (`value="3"`). I think it would be better to just expect one type and handle it, also warn user that there is a wrong prop type passed.

Reply:
It can be passed to the nouiSlder config as an array or a number, but If you think one type is better I remove that,